src/commands/interviews/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { availableDomains, getAvailableDomainsString } from '../../components/interview';
+
+export const validateDomainArg = (value: string): boolean | string => {
+  // validate if this is one of the available domains
+  if (value.toLowerCase() in availableDomains) return true;
+  return `you entered an invalid domain, please enter one of ${getAvailableDomainsString()}.`;
```

Comment:
English police has entered the chat. "Nu", the popo says.

Reply:
what's wrong with this, capitalizing the `you`?

Reply:
lol and split to 2 sentences

Reply:
I didn't capitalize because the bot replies like "@fishy, you entered an invalid domain, ..."